roles/testnode/vars/opensuse_leap_15.2.yml

```diff
@@ -2,18 +2,20 @@
 # vars specific to OpenSuse Leap 15.2
 packages_to_remove:
   - gettext-runtime-mini
+  - python
+  - python-base
 
 packages:
+  - python3-base
   - lsb-release
   - sysstat
   - gdb
   - make
+  - zypper
   - git
-#  - python-configobj
+  - python3-configobj
   # for running ceph
   - libedit0
-#  - libboost_thread1_54_0
-  - libboost_thread1_66_0
```

Comment:
Note: `libboost_thread1_66_0` is a dependency of `zypper`, so it *will* be installed on the nodes, but dropping it here since it's not needed for ceph.